lib/tests/streamlit/web/cli_test.py

```diff
@@ -111,7 +109,6 @@ def test_run_valid_url(self, temp_dir):
         with patch("validators.url", return_value=True), patch(
             "streamlit.web.cli._main_run"
         ), requests_mock.mock() as m:
-
```

Comment:
These empty lines are probably just autoformatter changes, right?

Probably we need to do something in the future (defiantly not right now :) ) to make all our python files match black requirements

Reply:
Yep, for whatever reason it wasn't conforming black